mldsa/sign.c

```diff
@@ -14,6 +14,76 @@
 #include "sign.h"
 #include "symmetric.h"
 
+static int mld_check_pct(uint8_t const pk[CRYPTO_PUBLICKEYBYTES],
+                         uint8_t const sk[CRYPTO_SECRETKEYBYTES]);
+
+#if defined(MLD_CONFIG_KEYGEN_PCT)
+/*************************************************
+ * [FIPS 140-3 IG]
+ * (https://csrc.nist.gov/csrc/media/Projects/cryptographic-module-validation-program/documents/fips%20140-3/FIPS%20140-3%20IG.pdf)
+ *
+ * VE10.35.02: Pair-wise Consistency Test (PCT) for DSA keypairs
```

Comment:
Should be TE rather than VE?

Reply:
Vendor Evidence vs Test Evidence. It's hard to say, no harm in adding both